### PR TITLE
Ovl_Bg_Ctower_Rot Ok

### PR DESCRIPTION
**Pull request by hensldm**
Ovl_Bg_Ctower_Rot take 2.

### CODE REVIEW

**Review by AngheloAlf — COMMENTED** [review]
Good job, just a minor comment

File: src/overlays/actors/ovl_Bg_Ctower_Rot/z_bg_ctower_rot.c [hunk 1]
```diff
@@ -21,22 +31,142 @@ const ActorInit Bg_Ctower_Rot_InitVars = {
...
+
+extern Gfx D_060174E0[];
+
+extern CollisionHeader D_06017650;
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
+};
+
+static Gfx* D_808E2564[] = { D_06012DA0, D_06017220, D_060174E0 };
```
Comment (Kenix3) [inline src/overlays/actors/ovl_Bg_Ctower_Rot/z_bg_ctower_rot.c:53]: `bgCtowerRotDlists`

**Review by Kenix3 — CHANGES_REQUESTED** [review]
One small variable left to name, otherwise it looks good.
Don't forget to run format.sh on this now that the repo can be formatted!

File: src/overlays/actors/ovl_Bg_Ctower_Rot/z_bg_ctower_rot.c (hunk 1 above)
Comment (hensldm) [reply src/overlays/actors/ovl_Bg_Ctower_Rot/z_bg_ctower_rot.c]: 👍

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**